issue.go

```diff
@@ -96,29 +96,58 @@ type Epic struct {
 	Done    bool   `json:"done" structs:"done"`
 }
 
+type Description struct {
```

Comment:
This type feels a bit superfluous to me. I think we could get rid of this and just have `Document` and `Content` to the same effect:

```go
type Content struct {
  Type    string    `json:"type,omitempty"`
  Text    string    `json:"text,omitempty"`
  Content []Content `json:"content,omitempty"`
}

type Document struct {
  Version int       `json:"version,omitempty"`
  Type    string    `json:"type,omitempty"`
  Content []Content `json:"content,omitempty"`
}
```